src/closure.js

```diff
@@ -5,17 +5,29 @@ const counter = () => {
   // Example: const newCounter = counter();
   // newCounter(); // 1
   // newCounter(); // 2
+  let count = 0;
+  return () => ++count;
 };
 
 const counterFactory = () => {
   // Return an object that has two methods called `increment` and `decrement`.
   // `increment` should increment a counter variable in closure scope and return it.
   // `decrement` should decrement the counter variable and return it.
+  let count = 0;
+  return {
+    increment() { return ++count; },
+    decrement() { return --count; }
+  };
 };
 
 const limitFunctionCallCount = (cb, n) => {
   // Should return a function that invokes `cb`.
   // The returned function should only allow `cb` to be invoked `n` times.
+  let count = 0;
+  return (...args) => {
+    if (count < n) { count++; return cb(...args); }
```

Comment:
Since you're performing multiple expressions here with `count++` and `return cb(...args)`, I would not one-line this. Break it out so that there is one expression per line.